test/RankifyInstance.ts

```
@@ -1266,9 +1262,16 @@ describe(scriptName, () => {
                   const setupResult = await allPlayersVotedTest(simulator)();
                   votes = setupResult.votes;
                 });
-                it('cannot end turn because players still have time to propose', async () => {
+                it.only('cannot end turn because players still have time to propose', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)